xml/cap_depl_install_caasp.xml

```diff
@@ -76,21 +76,48 @@
     <term>Network Requirements</term>
     <listitem>
      <para>
-      Your &kube; cluster needs its own domain and network. Each node should
-      resolve to its hostname, and to its fully-qualified domain name.
+      For &cf;'s nodes to be able to interact with each other, make sure your
+      network setup fulfills the following requirements:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        The &kube; cluster has an dedicated domain and subdomain.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Each node is able resolve to its host name and fully-qualified domain
```

Comment:
Looks like we've been using hostname as one word. It's probably better to keep things consistent.

Reply:
Please change the guide to using "host name" then: https://doc.opensuse.org/documentation/styleguide/app.terminology.html#sec.terminology

Reply:
"hostname" is the common usage pretty much everywhere.

Reply:
We do have style guide-defined spelling, however. You saying that *you think* everyone else is doing it differently than us really doesn't make a lot of difference there.

Reply:
It's not a matter of opinion, "hostname" is correct. If you really really want "host name" we'll change it in a separate commit.